climetlab/core/ipython.py

```diff
@@ -17,13 +17,12 @@
 from ..utils import module_loaded
 
 ipython_active = None
-if module_loaded("IPython"):
```

Comment:
This "if module_loaded..." avoid importing Ipython when it is not needed, we should keep it.

Reply:
Thanks, I review the climetlab.aaa.py ，I have find the changed code . Before the code be changed, the module_loaded function is not work as we proposal. So I remove the module_load function .
And  I readd it in the code.